✨ feat(cli): et clean (#980)
ethereum/execution-spec-tests: docs/CHANGELOG.md

- 🐞 Fix `Bytecode` class `__eq__` method ([#939](https://github.com/ethereum/execution-spec-tests/pull/939)).
- 🔀 Update `pydantic` from 2.8.2 to 2.9.2 ([#960](https://github.com/ethereum/execution-spec-tests/pull/960)).
- ✨ Add the `et make test` command, an interactive CLI that helps users create a new test module and function ([#950](https://github.com/ethereum/execution-spec-tests/pull/950)).

> - ✨ Add the `et clean` command to help clean generated files and directories from the repository ([#980](https://github.com/ethereum/execution-spec-tests/pull/980)).

Reviewer: ```suggestion
- ✨ Add the `et clean` command that helps delete generated files and directories from the repository ([#980](https://github.com/ethereum/execution-spec-tests/pull/980)).
```
